src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -171,16 +200,20 @@ public Tenant save(UUID consortiumId, UUID adminUserId, Tenant tenantDto) {
   @Override
   @Transactional
   public Tenant update(UUID consortiumId, String tenantId, Tenant tenantDto) {
-    consortiumService.checkConsortiumExistsOrThrow(consortiumId);
-    checkTenantExistsOrThrow(tenantId);
-    checkIdenticalOrThrow(tenantId, tenantDto.getId(), TENANTS_IDS_NOT_MATCHED_ERROR_MSG);
+    log.debug("update:: Trying to update tenant '{}' in consortium '{}'", tenantId, consortiumId);
+    var existedTenant = tenantRepository.findById(tenantId)
+      .orElseThrow(() -> new ResourceNotFoundException("id", tenantId));
+
+    validateTenantForUpdateOperation(consortiumId, tenantId, tenantDto, existedTenant);
+    // isDeleted flag cannot be changed by put request
+    tenantDto.setIsDeleted(existedTenant.getIsDeleted());
```

Comment:
After found issue relate to ablitiy of updating isCentral and isDeleted field, introduced improvement in order to prevent  future unwanted changes